test/array.spec.ts

```diff
@@ -5,8 +5,7 @@ const testFile: string = "array";
 
 test('it should create an array', () => {
     const instance: WebAssembly.Instance = getWasmInstanceSync(
-        path.join(__dirname, `${testFile}.tbs`),
-        path.join(__dirname, `bin/${testFile}.wasm`)
+        path.join(__dirname, `${testFile}.tbs`)
```

Comment:
I can change to this -> path.join(__dirname, 'array.tbs'). "__dirname" is mandatory otherwise path will be wrong when CWD change!